python/go_types.py

```diff
@@ -27,25 +24,15 @@ def uint8_array_to_numpy(go_array):
 
     # Wrap the buffer around to create a numpy array. Strangely, shape needs to be passed twice
     # This is a zero-copy operation
-    return np.ctypeslib.as_array(
-        (ctypes.c_uint8 * length).from_address(go_array.DataPtr), shape=shape
-    ).reshape(shape)
+    bytes_buffer = bytes(go.Slice_byte(go_array.Data))
```

Comment:
this is the new part, comes from a semi-recent PR on gopy, should make it more explicit what's the lifetime of data.